src/bindings/c/tests/ie_c_api_test.cpp

```diff
@@ -31,7 +31,11 @@ std::condition_variable condVar;
         std::string plugins_xml_std = TestDataHelpers::generate_ieclass_xml_path("plugins_win.xml");
     #endif
 #elif defined __APPLE__
+    #ifdef __aarch64__
+        std::string plugins_xml_std = TestDataHelpers::generate_ieclass_xml_path("plugins_apple_arm.xml");
```

Comment:
It need https://github.com/openvinotoolkit/testdata/pull/13 first merged.

Reply:
Can we register the plugin in the runtime without strange plugins xml files?

Reply:
@ilyachur The plugins xml files are the input parameter for api unite test "ie_core_create(plugins_xml, &core)", so I don't think we can register in runtime.

Reply:
If we don't modify plugins_apple.xml, let close testdata PR13?